### DISCUSSION TITLE
Creating a JLCPCB output for a Pannel

### DISCUSSION THREAD

**Discussion by alextrical** [INTI-CMNB/KiBot#715] [Q&A] [answered]
Im currently looking to create a panel with KiBot utilising KiKit's functionality, however im struggling to see how i can then convert that panel into a JLCPCB or Fabricatable file.

In KiKit i use the following command `kikit fab jlcpcb --no-drc --assembly --field LCSC --schematic circuit.kicad_sch circuit-panel.kicad_pcb KikitPanelJLCPCB` Though i cannot seem to find an equivalent option to trigger KiKit Fab in the guide for KiBot.

Any help is greatly appreciated

**Comment by set-soft** [comment]
You have a template for JLCPCB (and other houses too):  https://kibot.readthedocs.io/en/latest/configuration/imports.html#importing-internal-templates

**Reply by set-soft** [reply, marked as answer]
Note that this won't call KiKit. 
What KiKit fab does is already implemented on KiBot.
Also note that you must run the template when KiBot is called with the panel as input.
So you run KiBot once to generate the panel (and all the stuff related to the board) and then you run it again with the panel as input to generate things related to the panel.

**Reply by alextrical** [reply]
Ah, perfect. This morning, I was contemplating having to run the action again to process the generated panel, though i wasnt sure if this was best practice. Thank you very much for confirming that is the way to go about making the JLCPCB output.

It also makes sence why the project isnt making use of KiKit fab, as that is already taken care of by KiBot.
